x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/grok.test.ts

```diff
@@ -104,4 +104,26 @@ describe('Processor: Grok', () => {
       patterns: ['pattern1', 'pattern2', 'pattern3'],
     });
   });
+
+  test('accepts grok pattern that contains escaped characters', async () => {
+    const {
+      actions: { saveNewProcessor },
+      form,
+    } = testBed;
+
+    // Add "field" value
+    form.setInputValue('fieldNameField.input', 'test_grok_processor');
+
+    // Add the escaped value of \[%{HTTPDATE:timestamp}\]%{SPACE}\"%{WORD:http_method}%{SPACE}HTTP/%{NUMBER:http_version}\"
```

Comment:
nit: seems a bit redundant to me to write the whole escaped string in this comment

```suggestion
    // Add the escaped value into the input
```

Reply:
I hear you. It's definitely repetitive. TBH I liked having it because it saved me the trouble of having to read the escaped string, parse it for every backslash and then mentally unescape it. This was there's no mental computation necessary, the value is right there for me to read.